robottelo/helpers.py

```diff
@@ -366,6 +366,18 @@ def form_repo_path(org=None, lce=None, cv=None, cvv=None, prod=None, repo=None,
     return os.path.join(PULP_PUBLISHED_YUM_REPOS_PATH, repo_path)
 
 
+def form_repo_url(hostname=None, org=None, lce=None, cv=None, prod=None, repo=None):
```

Comment:
Please avoid referencing settings.server.hostname as a default, writing a new helper with this old pattern.

Make an instance of `satellite` a required kwarg, and use `satellite.url` property instead of `f'https://{hostname}'`

Reply:
We use this for both, Satellite and Capsule, so I'll pass the Capsule instead. Sat inherits from Caps anyway.

Reply:
ack - yep! The important part is using a Cap/Sat instance and its hostname property.